pkg/manifests/types.go

```diff
@@ -186,6 +195,13 @@ type PrometheusK8sConfig struct {
 	Tolerations []v1.Toleration `json:"tolerations,omitempty"`
 	// Defines the pod's topology spread constraints.
 	TopologySpreadConstraints []v1.TopologySpreadConstraint `json:"topologySpreadConstraints,omitempty"`
+	// Defines the scraping profile which Prometheus will use to scrape metrics
+	// from the platform components. Supported values are `full` or `minimal`.
+	// In the `full` profile (default) Prometheus will scrape all the metrics
+	// that are exposed by the different platform components. In the `minimal`
+	// profile Prometheus will only scrape metrics necessary for platform
+	// alerts, recording rules, telemetry and console dashboards.
+	ScrapeProfile ScrapeProfile `json:"scrapeProfile,omitempty"`
```

Comment:
Should it say collectionProfile rather?

Reply:
Right, because of the documentation, yes we should be consistent but then I'll change the name everywhere to collectionProfile to make it consistent all around. Let me know if you feel the opposite and we should only change the user-facing name

Reply:
:+1: